.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "typescript.tsdk": "node_modules\\typescript\\lib"
+  "typescript.tsdk": "node_modules\\typescript\\lib",
+  "workbench.editor.enablePreviewFromCodeNavigation": true
```

Comment:
This should be reverted, you'll want this to be a personal setting, not a workspace one.

Reply:
Apols, will revert